test/end-to-end/specs/api.test.js

```diff
@@ -3,7 +3,9 @@ const blueprintsPage = require("../pages/blueprints.page");
 
 // acceptable API response time
 // the 1 second comes from https://www.nngroup.com/articles/response-times-3-important-limits/
-const acceptable = 1000;
+// const acceptable = 1000;
```

Comment:
Do you mind taking out this line?

Reply:
I thought we might reinstate it once OSCI moves to better infra, but I don't mind taking it out and adjusting the comment.

Reply:
Updated